README.md

```diff
@@ -23,6 +23,14 @@ View the file through the 'index.html'.
 - Twitter: [@twitterhandle](https://twitter.com/tahseen1zahra)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/tahseenzahra/)
 
+👤 **Author**
```

Comment:
- [ Optional ] You should use the actual name of the author on behalf of the placeholder text (**Author**)